resources/definitions/deltacomb.def.json

```diff
@@ -25,41 +25,32 @@
         "machine_start_gcode":                   { "default_value": "G21 ;metric values\nG90 ;absolute positioning\nM82 ;set extruder to absolute mode\nM107 ;start with the fan off\nG28 ;Home all axes (max endstops)\nG1 Z15.0 F9000 ;move the platform down 15mm\nG92 E0 ;zero the extruded length\nG1 F200 E3 ;extrude 3mm of feed stock\nG92 E0 ;zero the extruded length again\nG1 F9000\n;Put printing message on LCD screen\nM117 Printing..."},
         "machine_end_gcode":                     { "default_value": "M104 S0 ;extruder heater off\nM140 S0 ;heated bed heater off (if you have it)\nG91 ;relative positioning\nG1 E-1 F300  ;retract the filament a bit before lifting the nozzle, to release some of the pressure\nG28 ;Home all axes (max endstops)\nM84 ;steppers off\nG90 ;absolute positioning" },
         "machine_shape":                         { "default_value": "elliptic" },
-        "machine_max_feedrate_x":                { "default_value": 250 },
-        "machine_max_feedrate_y":                { "default_value": 250 },
-        "machine_max_feedrate_z":                { "default_value": 15 },
-        "machine_max_acceleration_x":            { "default_value": 10000 },
-        "machine_max_acceleration_y":            { "default_value": 10000 },
-        "machine_max_acceleration_z":            { "default_value": 50 },
-        "machine_max_acceleration_e":            { "default_value": 100 },
-        "machine_acceleration":                  { "default_value": 4000 },
-        "machine_max_jerk_xy":                   { "default_value": 25.0 },
-        "machine_max_jerk_z":                    { "default_value": 0.4 },
-        "machine_max_jerk_e":                    { "default_value": 1.0 },
-        "retraction_hop_enabled":                { "default_value": false },
-        "retraction_amount" :                    { "default_value": 4.5 },
-        "retraction_speed" :                     { "default_value": 40 },
+        "retraction_hop_enabled":                { "default_value": true },
+        "retraction_amount" :                    { "default_value": 3.5 },
+        "retraction_speed" :                     { "default_value": 50 },
         "material_diameter":                     { "default_value": 1.75 },
         "material_final_print_temperature":      { "value": "material_print_temperature - 5" },
         "material_initial_print_temperature":    { "value": "material_print_temperature" },
         "material_print_temperature_layer_0":    { "value": "material_print_temperature + 5" },
         "travel_avoid_distance":                 { "default_value": 1, "value": 1 },
-        "speed_print" :                          { "default_value": 70 },
-        "speed_travel":                          { "default_value": 80, "value": 80 },
+        "speed_print" :                          { "default_value": 60 },
+        "speed_travel":                          { "value": 150.0 },
```

Comment:
Please not that all `value`s must have double-quotes around, so here it should be "150.0"